package/network/utils/xdp-tools/Makefile

```diff
@@ -2,16 +2,17 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=xdp-tools
 PKG_RELEASE:=1
-PKG_VERSION:=1.4.3
-PKG_HASH:=6e727e2c7c079dec86575c5d0e7aac7ecd582aabe04b3b176d7ef50e653b60c0
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://codeload.github.com/xdp-project/xdp-tools/tar.gz/v$(PKG_VERSION)?
-PKG_ABI_VERSION:=$(call abi_version_str,$(PKG_VERSION))
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/xdp-project/xdp-tools.git
+PKG_SOURCE_DATE:=2025-11-24
+PKG_SOURCE_VERSION:=d37286ad6ebb5519b2f33cccfc36507ef8e10f5f
+PKG_MIRROR_HASH:=b9f30beacda20a7595855ca06ec2b6fefb688f2e94804a9c5d78875833c250a0
+PKG_ABI_VERSION:=$(call abi_version_str,$(PKG_SOURCE_DATE))
 
 PKG_MAINTAINER:=Daniel Golle <daniel@makrotopia.org>
 
-PKG_BUILD_DEPENDS:=bpf-headers
+PKG_BUILD_DEPENDS:=bpf-headers bpftool/host
 PKG_FLAGS:=nonshared
```

Comment:
Is `nonshared` still needed? Does this depend on the target?

Reply:
I think you still have different endianess?